articles/migrations/index.md

```diff
@@ -25,6 +25,85 @@ If you need help with the migration, create a ticket in our [Support Center](${e
 
 Current migrations are listed below, newest first. For migrations that have already been enabled see [Past Migrations](#past-migrations).
 
+### Introducing Lock v11, Auth0.js v9
+
+| Severity | Grace Period Start | Mandatory Opt-In|
+| --- | --- | --- |
+| Medium | 2017-12-21 |  2018-04-01 |
+
+We’re continually improving the security of our service. As part of this, we are deprecating a set of APIs (`usernamepassword/login`, `/ssodata`, `/tokeninfo`, `/delegation`) used by Lock.js v8, v9, and v10 and and auth0.js, v6, v7, and v8. You should update your applications by **April 1, 2018**.
+
+We are offering two options for this update:
+
+1. We are moving towards a centralized login experience and are recommending this option moving forward. There are many benefits of using a centralized login experience, read more about how Centralized Login works [here](/hosted-pages/login). Moving to the centralized login experience is straightforward and will simplify your code base. Read [this guide](/guides/login/centralized-vs-embedded) to learn more about the benefits of centralized login vs embedded login.
+1. If you decide you need to continue using embedded login, here are the migration guides for [Auth0.js](/libraries/auth0js/v9/migration-guide) and [Lock](/libraries/lock/v11/migration-guide).
+
+#### Am I affected by the change?
+
+If you are currently using Lock v9 or v10, or Auth0.js v7 or v8, you will be affected.
+
+If you have any questions, create a ticket in our [Support Center](${env.DOMAIN_URL_SUPPORT}).
+
+### Introducing Resource Owner Support for oauth/token Endpoint
+
+| Severity | Grace Period Start | Mandatory Opt-In|
+| --- | --- | --- |
+| Medium | 2017-12-21 |  2018-07-01 |
+
+We’ve introduced support for Resource Owner Password to the `oauth/token` endpoint earlier this year ([doc](/api/authentication#resource-owner-password)). We will deprecate the current `oauth/ro` and `oauth/access_token` endpoints. Applications must be updated before July 1, 2018, when `oauth/ro` will become unavailable. A migration guide will be available in Q1.
+
+#### Am I affected by the change?
+
+If you are currently implementing the `oauth/ro` endpoint your application will need to be updated.
+
+If you have any questions, create a ticket in our [Support Center](${env.DOMAIN_URL_SUPPORT}).
+
+### API Authorization with 3rd Party Vendor APIs. Deprecating Delegation Endpoint
+
+| Severity | Grace Period Start | Mandatory Opt-In|
+| --- | --- | --- |
+| Medium | 2017-12-21 |  2018-06-01 |
+
+We’re re-architecting the mechanism by which you get tokens for 3rd Party/Vendor APIs (such as AWS, Firebase, and others). It now works the same as any custom API, providing better consistency. This new architecture will be available in Q1 2018 and its availability will officially deprecate the Delegation Endpoint. All clients must be updated prior to June 1, 2018. Formal migration guide will be available in Q1.
```

Comment:
- `We're` does not look good, let's not use it. We are.
- `We’re re-architecting the mechanism by which you get tokens for 3rd Party/Vendor APIs` -> `We are changing how you get tokens for 3rd Party/Vendor APIs`
- `such as` -> `for example,`
- `It now works`? It's not shipped so it doesn't work now like that
- `This new architecture will be available in Q1 2018 and its availability will officially deprecate` -> `This will be available in Q1 of 2018, when the Delegation Endpoint will be officially deprecated`
- add link to delegation endpoint
- `Formal` migration guide? `The migration guide...`